pravega/pravega-operator
# Issue 203: clean up webhook k8s svc

### Change log description

This PR adds the ownerreference to the webhook k8s service. This change will let the k8s service to be garbage collected by k8s when the operator deployment is deleted.

### Purpose of the change

Fix #203 

### What the code does

Instead of using the `BootstrapOption` to create the k8s service for webhook, the code manually creates one and add the operator deployment to be its owner, so that it could be garbage collected.

### How to verify it

Create and delete the pravega operator, the webhook k8s service should also be deleted. Use `kubectl get svc` to verify.
